vaadin-grid-flow-parent/vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/GridSortingTest.java

```diff
@@ -293,6 +293,27 @@ public void setMultiSortWithPriorityNull_throws() {
         grid.setMultiSort(true, null);
     }
 
+    @Test
+    public void setMultiSortShiftClickOnly() {
+        grid.setMultiSort(true, true);
+
```

Comment:
Please also check `grid.getElement().hasAttribute("multi-sort-on-shift-click");` (or rather the `multiSortOnShiftClick` property).

An additional test could check the case when the attribute is not there and `isMultiSortShiftClickOnly` returns false.

Reply:
Done.